src/xmipp/applications/programs/mpi_nma_alignment_vol/mpi_nma_alignment_vol_main.cpp

```diff
@@ -0,0 +1,140 @@
+/***************************************************************************
+ *
+ * Authors:  Slavica Jonic slavica.jonic@impmc.jussieu.fr
+ *           Carlos Oscar Sanchez Sorzano coss.eps@ceu.es
+ *
+ * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+ * Lab. de Bioingenieria, Univ. San Pablo CEU
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+ * 02111-1307  USA
+ *
+ *  All comments concerning this program package may be sent to the
+ *  e-mail address 'xmipp@cnb.uam.es'
+ ***************************************************************************/
+//#include <mpi.h>
+#include <parallel/xmipp_mpi.h>
+#include <reconstruction/nma_alignment_vol.h>
+
+
+/** Class to perfom the NMA Alignment with  MPI parallelization */
+class MpiProgNMAVol: public ProgNmaAlignmentVol
+{
+private:
+    MpiNode *node;
```

Comment:
follow[ rule of 5 or 0](https://en.cppreference.com/w/cpp/language/rule_of_three)